### ISSUE TITLE
Cannot installable

### ISSUE THREAD

**Issue by 7system7** [brpaz/ulauncher-raindrop#2]
Sadly, when I try to add this extension, it throws this:

```
Jun 14 13:11:08 AMBER ulauncher.desktop[928089]: file:///usr/share/ulauncher/preferences/dist/static/js/app.93702a5ce089a3ccbf36.js:4919:18: CONSOLE ERROR TypeError: e.cancel is not a function. (In 'e.cancel()', 'e.cancel' is undefined)
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]: 2021-06-14 13:11:09,060 | #033[1;31mERROR#033[0m | ulauncher.ui.windows.PreferencesUlauncherDialog: on_scheme_callback() | Unexpected API error. HTTPError: HTTP Error 422: Unprocessable Entity
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]: Traceback (most recent call last):
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:   File "/usr/lib/python3/dist-packages/ulauncher/ui/windows/PreferencesUlauncherDialog.py", line 198, in on_scheme_callback
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:     resp = rt.dispatch(self, scheme_request.get_uri())
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:   File "/usr/lib/python3/dist-packages/ulauncher/utils/Router.py", line 49, in dispatch
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:     return callback(context, url_params)
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:   File "/usr/lib/python3/dist-packages/ulauncher/ui/windows/PreferencesUlauncherDialog.py", line 433, in prefs_extension_add
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:     ext_id = downloader.download(url)
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:   File "/usr/lib/python3/dist-packages/ulauncher/api/server/ExtensionDownloader.py", line 75, in download
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:     commit = gh_ext.find_compatible_version()
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:   File "/usr/lib/python3/dist-packages/ulauncher/api/server/GithubExtension.py", line 80, in find_compatible_version
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:     return self.get_commit(sha_or_branch)
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:   File "/usr/lib/python3/dist-packages/ulauncher/api/server/GithubExtension.py", line 89, in get_commit
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:     resp = json.loads(urlopen(url).read().decode('utf-8'))
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:   File "/usr/lib/python3.9/urllib/request.py", line 214, in urlopen
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:     return opener.open(url, data, timeout)
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:   File "/usr/lib/python3.9/urllib/request.py", line 523, in open
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:     response = meth(req, response)
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:   File "/usr/lib/python3.9/urllib/request.py", line 632, in http_response
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:     response = self.parent.error(
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:   File "/usr/lib/python3.9/urllib/request.py", line 561, in error
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:     return self._call_chain(*args)
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:   File "/usr/lib/python3.9/urllib/request.py", line 494, in _call_chain
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:     result = func(*args)
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:   File "/usr/lib/python3.9/urllib/request.py", line 641, in http_error_default
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]:     raise HTTPError(req.full_url, code, msg, hdrs, fp)
Jun 14 13:11:09 AMBER ulauncher.desktop[6226]: urllib.error.HTTPError: HTTP Error 422: Unprocessable Entity
```

Machine:

```bash
$ sudo inxi -bz --memory
System:    Kernel: 5.11.0-18-generic x86_64 bits: 64 Desktop: GNOME 3.38.4 Distro: Ubuntu 21.04 (Hirsute Hippo) 
Machine:   Type: Laptop System: LENOVO product: 20QV00CEGE v: ThinkPad X1 Extreme 2nd serial: <filter> 
           Mobo: LENOVO model: 20QV00CEGE v: SDK0T08861 WIN serial: <filter> UEFI: LENOVO v: N2OET51W (1.38 ) date: 02/18/2021 
Battery:   ID-1: BAT0 charge: 75.9 Wh condition: 75.9/80.4 Wh (94%) 
Memory:    RAM: total: 62.62 GiB used: 13.75 GiB (22.0%) 
           Array-1: capacity: 128 GiB note: est. slots: 4 EC: None 
           Device-1: ChannelA-DIMM0 size: 32 GiB speed: 2667 MT/s 
           Device-2: ChannelB-DIMM0 size: 32 GiB speed: 2667 MT/s 
CPU:       Info: 8-Core Intel Core i9-9880H [MT MCP] speed: 800 MHz min/max: 800/4800 MHz 
Graphics:  Device-1: NVIDIA TU117M [GeForce GTX 1650 Mobile / Max-Q] driver: nvidia v: 460.80 
           Device-2: Chicony Integrated Camera type: USB driver: uvcvideo 
           Display: server: X.Org 1.20.11 driver: loaded: nvidia resolution: 2560x1080 
           OpenGL: renderer: GeForce GTX 1650/PCIe/SSE2 v: 4.6.0 NVIDIA 460.80 
Network:   Device-1: Intel Ethernet I219-LM driver: e1000e 
           Device-2: Intel Wi-Fi 6 AX200 driver: iwlwifi 
Drives:    Local Storage: total: 2.79 TiB used: 2.39 TiB (85.6%) 
Info:      Processes: 628 Uptime: 2d 18h 51m Shell: Zsh inxi: 3.3.01

**Comment by joekaiser** [comment]
I have the same problem. See https://github.com/Ulauncher/Ulauncher/issues/695  The wrong version `master` is specified in the version file. This repo is using `main`

I will Submit a PR

**Comment by brpaz** [comment]
Fixed in #3
